.travis.yml

```diff
@@ -46,27 +49,32 @@ jobs:
         - (cd web && npm run build:prod)
           && mvn clean install -q -f lib -DskipTests -Dmaven.javadoc.skip=true
           && mvn clean install -q -f rest -DskipTests -Dmaven.javadoc.skip=true
-    - stage: "Release / Deploy"
-      script: 
-        - ci/change_version.sh -i -m "lib/ rest/" -swarm lib/taskana-cdi/src/test/java/pro/taskana/TaskanaProducersTest.java
-          && ci/copy-rest-spring.sh $TRAVIS_TAG
-          && ci/commitPoms.sh lib/taskana-cdi/src/test/java/pro/taskana/TaskanaProducersTest.java
-      deploy: 
-        provider: cloudfoundry 
-        username: tobias.schaefer@novatec-gmbh.de
-        password:
-          secure: dQh3e8Zhi2ucd5NcxHZ337Ku2SFrt0qA8lghtExQU5bcdFziqYKJsVFdD19hNQE6DOZT8FV9OHTB3odZSvV7aa38sq4KDwzJVVitIyU3eNZP0S5AqAv0eKitqQhjY7rnjBIkgMQ2rGTZlAL9g/CQWPlB4rnUPLXv/7+pZP29M3OPK0hyIPw9I0FfU02cM+R2z8SYPRr4MYYbdsTdJeJz+1sKWwQiD4bDE/Kmcs7BALTZ3/HFmu2Kjxk7soERSpXlMvKDkHHJCJMSjQI6zT5Ja6yg4SgOgccLZKuaPwF+8a7RKPOtW+H74zzO/IE9FGimlRKFGHhoFmQh0oI1sMkhRyIH0n/rkCVzI6B8XjDBEdaRjfHZbAkrjapuvhLwSxyklut3CknbSxfgAo8tZefsGVETD1rYkfLA207bkxt6Wk+X1d2pCRFkUoeTjcB+IkqvftTJvTX5vyt927w0pyx/xg9gxBLA3bqzhRL7rwgXgqgrxNUDFT3nxExecPpdEinyVf+jQxmrb1uw1WEFM+dLjA3tByxMEy7PfnfqTvUs8dvs4em1Vln+N5K1+OqU93ze3FmPMmqY3/4r63qdVACtCqHlfGC3HQPUi6fWmBzVNhZ9rF6FJvkECTkvLklnOB9MIoneGk2v46slveUsxKSz985A2aMkYNO+p1VozS8rAfg=
-        api: https://api.ng.bluemix.net 
-        organization: '"NovaTec Consulting GmbH"' 
-        space: Taskana
-        on:
-          all_branches: true
-      if: repo = env(DEPLOY_REPO) AND tag =~ ^v[0-9]+\.[0-9]+\.[0-9]+$
-    - stage: "Release / Deploy"
+    - stage: Release
       script:
         - ci/release.sh lib/taskana-core $TRAVIS_TAG
           && ci/release.sh lib/taskana-spring $TRAVIS_TAG
           && ci/release.sh lib/taskana-cdi $TRAVIS_TAG
           && ci/release.sh web/ $TRAVIS_TAG
           && ci/release.sh rest/taskana-rest-spring $TRAVIS_TAG
       if: repo = env(DEPLOY_REPO) AND (tag =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ OR branch = master) AND type != pull_request
+    - stage: AfterRelease
+      script: 
+        - ci/change_version.sh -i -m "lib/ rest/" -swarm lib/taskana-cdi/src/test/java/pro/taskana/TaskanaProducersTest.java
+          && ci/commitPoms.sh lib/taskana-cdi/src/test/java/pro/taskana/TaskanaProducersTest.java
+      if: repo = env(DEPLOY_REPO) AND tag =~ ^v[0-9]+\.[0-9]+\.[0-9]+$
+    - stage: Deploy
+      script: 
+        - if [[ $FORCE_DEPLOY_TAG =~ ^v[0-9]+\.[0-9]+\.[0-9] ]];
```

Comment:
With the changes from the first commit i'd modify the `copy-rest-spring.sh` file to use two parameters. The tag and force deploy.

Reply:
Ok, makes sense